frame/fast-unstake/src/lib.rs

```diff
@@ -15,47 +15,100 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! A pallet that's designed to JUST do the following:
+//! # Fast Unstake Pallet
```

Comment:
The docs are missing some headings from https://hackmd.io/4pjRqPNNTnGXJHKK9jv90A. Is the hackmd still up to date?

Reply:
Which ones? Some are optional, and I may have made some last minute changes 🙈

Reply:
`### Design Goals` and `### Design` from `## Low Level Implementation Details`